Radzen.Blazor/RadzenDataGridColumn.razor.cs

```diff
@@ -323,6 +323,14 @@ public string ColumnPickerTitle
         [Parameter]
         public string CustomFilterExpression { get; set; }
 
+        /// <summary>
+        /// Gets or sets ClearCustomFilter.
+        /// ClearCustomFilter should reset all data related to filter.
+        /// ClearCustomFilter is invoked in <see cref="ClearCustomFilterAsync"/>.
+        /// </summary>
+        [Parameter]
+        public EventCallback ClearCustomFilter { get; set; }
```

Comment:
There is already FilterCleared event

Reply:
Ok, I did sth different.
Problem is that it won't fix problem of method RadzenDataGridColumn.ClearFilters(). But it solves my problem so it's suitable solution for me.

I would suggest you to add bug to your backlog to fix CustomFilterExpression problem in the future.